delfin/tests/unit/drivers/dell_emc/unity/test_emc_unity.py

```diff
@@ -597,3 +596,12 @@ def test_err_storage_pools(self):
             self.driver.list_storage_pools(context)
         self.assertIn('Bad response from server',
                       str(exc.exception))
+
+    def test_clear_alert(self):
+        alert_id = 101
+        m = mock.MagicMock(status_code=200)
+        with mock.patch.object(Session, 'delete', return_value=m):
+            m.raise_for_status.return_value = 200
+            m.json.return_value = None
+            re = self.driver.clear_alert(context, alert_id)
```

Comment:
This seem can not test the api, for example, if I modify clear_alert to this:
```
def clear_alert(self, context, alert_id):
    return None
```
This unit test also can be run successfully

Reply:
Same for the line 591
Please check all other unit test to make sure that the unit test would failed when we do some unexpected changes

Reply:
done